news-bundle/src/Resources/contao/modules/ModuleNews.php

```diff
@@ -148,12 +148,12 @@ protected function parseArticle($objArticle, $blnAddArchive=false, $strClass='',
 		$arrMeta = $this->getMetaFields($objArticle);
```

Comment:
Looks like `getMetaFields` should always return all properties (`date` and `author` in your case) as empty strings, instead of checking for null-values.

Reply:
I pushed a new commit. Do you mean like this? Wouldn’t this be a behaviour change, since templates might treat empty string authors and `null` authors differently?

Reply:
Solved differently based on review by @leofeyer.